### ISSUE TITLE
Can't update Number voice status callback

### ISSUE THREAD

**Issue by nimo55pl** [Vonage/vonage-php-sdk-core#271]
Can't update Numer voiceStatusCallbackUrl

```
$number = $client->numbers()->get(NUMBER);
$number->setVoiceDestination(FORWARD_NUMBER, 'tel');
$number->setWebhook(\Vonage\Numbers\Number::WEBHOOK_VOICE_STATUS, 'https://example.com/callback');
```
`\Vonage\Numbers\Number::WEBHOOK_VOICE_STATUS = 'voiceStatusCallbackUrl'` is wrong, it works when I changed it to `\Vonage\Numbers\Number::WEBHOOK_VOICE_STATUS = 'voiceStatusCallback'`

**Comment by marphi** [comment]
I have the same issue. Vonage?Nexmo are you still support this SDK? This issue has been reported 7 months ago...

**Comment by SecondeJK** [comment]
@marphi @nimo55pl I am now maintaining this library, so now that v3 has been released will look to fix this in a minor release as soon as I can

**Comment by SecondeJK** [comment]
See #315

**Comment by SecondeJK** [comment]
Fixed by release 3.0.5
